README.md

```diff
@@ -112,7 +112,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check out code
-        uses: actions/checkout@v1
+        uses: actions/checkout@v3.5.2
```

Comment:
Why not use v3?

```suggestion
        uses: actions/checkout@v3
```

Reply:
I think that fixing the version would reduce the possibility that CI would suddenly fail due to the release of a new version.
However, specifying by major version is a common method, so I will adopt that method.

Reply:
I fix it: v